client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -86,6 +86,12 @@ export default FormBase.extend({
                     .append(this._footer())
             );
         }
+        options.tool_errors &&
+            this.message.update({
+                status: 'warning',
```

Comment:
maybe we could use `error` to be more prominent

Reply:
Might be alert, error didn't do anything

Reply:
it is probably based on bootstrap, so try`danger`

Reply:
That was it, thanks @martenson !
<img width="309" alt="screen shot 2018-09-08 at 17 58 48" src="https://user-images.githubusercontent.com/6804901/45256023-e301ab00-b390-11e8-8786-99b2b95bfc26.png">

Reply:
I've rebased the last commit with this change.